assets/src/block-templates/issues/template.js

```diff
@@ -0,0 +1,112 @@
+import mainThemeUrl from '../main-theme-url';
+
+const item = ['core/group', {
+  backgroundColor: 'white',
+  style: {
+    border: { radius:'4px' },
+    spacing: {
+      padding: {
+        top:'32px',
+        right:'16px',
+        bottom:'32px',
+        left:'16px',
+      }
+    },
+  },
+  layout: {
```

Comment:
I don't see these layout settings in the sidebar after adding the pattern, do you know where they can be edited?

Reply:
Do you mean this?
![Screenshot 2023-01-16 at 13 41 39](https://user-images.githubusercontent.com/77975803/212728543-9f6388b6-4918-4043-ab85-e13f092cb0f1.png)

Reply:
Aaaaah yes, ok it's only for Row blocks! Thanks 👍